index-de.html

```diff
@@ -195,7 +200,7 @@ <h3 id="commit">Commit</h3>
   <p>Ein vierter Fall ist, ein Commit mit <a href="#detached">detached HEAD</a>,
   wird weiter unten erklärt.</p>
 
-  <h3 id="checkout">Checkout</h3>
+  <h3 id="checkout">Checkout (Ausprobieren)</h3>
```

Comment:
@HybridDog Now I got it, exiting. Just brainstorming... What do you think about "Aus dem Repository laden". It's a bit lengthy, I know, but I do think it covers most `checkout` use cases, namely switching branches, checking out previous commits and also loading single files from other commits/branches/tags.